.github/workflows/test.yaml

```diff
@@ -24,35 +24,42 @@ jobs:
           node-version: ${{ matrix.node-version }}
           registry-url: https://registry.npmjs.org/
 
-      - name: Restore cache of node modules.
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn config get cacheFolder)"
+
+      - name: Restore cache of node modules and yarn cache
         uses: actions/cache@v2
         with:
           path: |
             node_modules
             */*/node_modules
-          key: ${{ runner.os }}-lerna-${{ hashFiles('**/yarn.lock') }}
+            ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-nodejs-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-nodejs-
 
       - name: Install dependencies
         run: |
           yarn --prefer-offline
-          node_modules/.bin/lerna bootstrap
+          yarn lerna bootstrap
 
       - name: Build (all)
-        if: matrix.package == 'all' 
+        if: matrix.package == 'all'
         run: |
-          node_modules/.bin/lerna link 
-          node_modules/.bin/lerna run build --include-dependencies --ignore @hoprnet/hopr-core --ignore @hoprnet/hopr-ethereum --ignore @hoprnet/hopr-core-ethereum
+          yarn lerna link
```

Comment:
Same here, this is a `lerna` command, not a script from our `package.json`

Reply:
yarn will look for the lerna binary in `node_modules/.bin` and use ut

Reply:
Great, check my previous comment, using `npx` should also be fine